src/Avalonia.Themes.Default/Controls/CheckBox.xaml

```diff
@@ -26,6 +26,7 @@
                     Stretch="Uniform"
                     HorizontalAlignment="Center"
                     VerticalAlignment="Center"
+                    FlowDirection="LeftToRight"
```

Comment:
Why do you set the FlowDirection explicitly? This disables inheritance.

Reply:
According to Microsoft's documentation, `Path` is supposed to inherit `FlowDirection` as you wrote.

The V-glyph should always appear not in mirror view.